src/main/kotlin/com/autonomousapps/advice/Advice.kt

```diff
@@ -87,13 +92,23 @@ data class Advice(
   fun isAdd() = fromConfiguration == null && !isCompileOnly()
 
   /**
-   * An advice is "remove-advice" if it is declared and not used, AND is not `compileOnly`.
+   * An advice is "remove-advice" if it is declared and not used, AND is not `compileOnly`,
+   * AND is not `processor`.
    */
-  fun isRemove() = toConfiguration == null && !isCompileOnly()
+  fun isRemove() = toConfiguration == null && !isCompileOnly() &&!isProcessor()
 
   /**
    * An advice is "change-advice" if it is declared and used (but is on the wrong configuration),
    * AND is not `compileOnly`.
    */
   fun isChange() = fromConfiguration != null && toConfiguration != null && !isCompileOnly()
+
+  /**
+   * An advice is "processors-advice" if it is declared on a k/apt
+   * or annotationProcessor configuration.
+   */
+  fun isProcessor() = fromConfiguration != null &&
```

Comment:
I think that `fromConfiguration != null` is redundant, given lines 112-113, where you test that `fromConfiguration` ends with kapt or annotationProcessor. You might need to do this:
```kotlin
fromConfiguration?.endsWith("kapt", ignoreCase = true) == true
```

Reply:
I didn't like the 2 `?.`, so I used a let. Let me know if it's ok for you.